### DISCUSSION TITLE
Header component renders nav element instead of header

### DISCUSSION THREAD

**Discussion by vincentole** [mantinedev/mantine#1144] [Feedback]
Hello everyone, 

first of all thanks for the great work. I really like mantine.

However, I am very confused why the `Header` component renders as `nav` and not as `header` as one would expect. 
As far as I could figure out, it also doesn't accept `components` to change the behavior. Generally, docs on `Header` would really help. 

Is there any chance that this could be changed. And could someone explain the reasoning behind this decision?

Thanks you very much and have a great day :)

**Comment by rtivital** [comment]
In most cases you would use Header component to display navigation, that is why it is rendered as nav. Examples of headers – https://ui.mantine.dev/category/headers

**Reply by vincentole** [reply]
This is true, but generally you would have a `header` tag  and inside maybe no nav (if nav is a sidebar) or a logo, nav and social nav. In those cases it would be great to semantically differentiate those. 

Is there a way I could use the Header component in this kind of way? (without any nav, or with a header as root and children as nav)

Thanks for your help!

**Reply by rtivital** [reply]
No, currently it is not supported

**Reply by vincentole** [reply]
Alright, thanks for your explanation though.
